examples/generative/vq_vae.py

```diff
@@ -1,15 +1,15 @@
 """
 Title: Vector-Quantized Variational Autoencoders
-Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Author: [Sayak Paul](https://twitter.com/RisingSayak), [Rein van 't Veer](https://github.com/reinvantveer)
```

Comment:
Given the edits are small and there were bare-minimum contributions to the code, I don't feel comfortable with this edit.

Reply:
Please bear in mind that I also contributed #911 and #921. These were small edits as well, but the do stack up. If you still feel uncomfortable with attaching my name, please let me know and I will revert that commit

Reply:
But I'm happy with an acknowledgment to call out your contribution from the tutorial.

Reply:
The tutorials I have co-authored on keras.io share the following:

* All the authors contributed significant portions of the code. 
* All the authors were thoroughly involved in structuring the tutorials.
* All the authors contributed significant portions of the written parts. 

I am tagging my fellow co-authors for reference: @ariG23498, @soumik12345.

This PR deviates from that. So, as a courtesy to honor your contributions, I am absolutely comfortable with an acknowledgment but not with a co-authorship status, sorry.

Reply:
Totally fine @sayakpaul I understand, thanks for explaining